CHANGELOG.rst

```diff
@@ -188,13 +214,32 @@ Improvements to the `pysys.py` command line tool:
   case insensitively) in their `id` or `title`. This can be a convenient way 
   to quickly run a set of tests related to a particular feature area.  
 
+- Added a concise summary of the test ids for any non-passes in a format that's 
+  easy to copy and paste into a new command, such as for re-running the failed 
+  tests. This can be disabled using the `ConsoleSummaryResultsWriter` property 
+  `showTestIdList` if desired. 
+
+- Added an environment variable PYSYS_DEFAULT_THREADS which can be used to set 
+  the number of threads to use with `--threads auto` is specified on a 
+  per-machine or per-user basis. 
+
 - Added `--json` output mode to `pysys.py print` which dumps full information 
   about the available tests in JSON format suitable for reading in from other 
   programs. 
+
+- Change `makeproject` so that templates must be specified with an explicit 
+  argument `--template=NAME`. 
   
 
 Upgrade guide and compatibility:
 
+- `ProcessUser.mkdir` now returns the absolutized path (including the output 
```

Comment:
Just use "absolute" here, "absolutized" is not a real word.

Reply:
yes